src/xo-app/vm/tab-network.js

```diff
@@ -405,6 +468,9 @@ class NewVif extends BaseComponent {
   }
 }
 
+@connectStore(() => ({
+  vifs: createGetObjectsOfType('VIF').pick((state, props) => props.vm.VIFs),
```

Comment:
For better readability, we usually mark a parameter as `_` if we don't use it (you're not using `state`).